nifs/atomvm_ssd1306.c

```diff
@@ -215,6 +232,43 @@ static esp_err_t write_page_command(i2c_port_t i2c_num, uint8_t *page, uint8_t p
     return err;
 }
 
+static term nif_ssd1306_set_inversion(Context *ctx, int argc, term argv[])
+{
+    TRACE(TAG ": nif_ssd1306_set_inversion\n");
+    UNUSED(argc);
+
+    term i2c_num_term = argv[0];
+    VALIDATE_VALUE(i2c_num_term, term_is_atom);
+    term mode_term = argv[1];
+    VALIDATE_VALUE(i2c_num_term, term_is_atom);
+
+    i2c_port_t i2c_num = get_i2c_port_num(ctx->global, i2c_num_term);
+    if (i2c_num == I2C_NUM_MAX) {
+        ESP_LOGE(TAG, "Invalid I2C portnum");
+        RAISE_ERROR(BADARG_ATOM);
+    }
+
+    bool inversion_mode;
+    switch(term_to_atom_index(mode_term)) {
+      case TRUE_ATOM_INDEX:
+        inversion_mode = true;
+        break;
+      case FALSE_ATOM_INDEX:
+        inversion_mode = false;
+        break;
+      default:
+        ESP_LOGE(TAG, "Invalid inversion mode");
+        RAISE_ERROR(BADARG_ATOM);
+    }
+
+    esp_err_t err = set_display_inverted_command(i2c_num, inversion_mode);
+    if (err != ESP_OK) {
```

Comment:
Maybe we should raise an error here like other places in the driver that pass the parameter validation, but the final operation fails. Otherwise we are returning `ok`, even though the operation failed.

What we **really** should be doing in these places is returning {error, Reason}, but those changes require delicate care, and should be done together in a separate PR.

Reply:
Thanks for the feedback.
I did change it to return `ERROR_ATOM` and log the error code returned from i2c.

Returning proper `{error, Reason}` sounds like too much for me for now, because I don't know how to build tuples like this, and how to encode reason in proper format.